config/config.go

```diff
@@ -117,5 +119,12 @@ func Parse(in *MountParams) (*MountConfig, error) {
 		return nil, fmt.Errorf("failed to unmarshal secrets attribute: %v", err)
 	}
 
+	for i := range out.Secrets {
+		name := out.Secrets[i].FileName
+		if errs := validation.IsConfigMapKey(name); len(errs) != 0 {
+			return nil, fmt.Errorf("%q is not a valid fileName for Secret: %s", name, strings.Join(errs, ";"))
```

Comment:
Should we document the supported charset: `[-._a-zA-Z0-9]+` + under 253 chars somewhere? A GCP secret ID can be up to 255, so this may be an inconvenience for some users with long secret names that want their filename to match their secret ID.

Reply:
Updated the changelog to record this